src/app/shared/components/input-google-autocomplete/input-google-autocomplete.component.spec.ts

```
@@ -1,70 +1,214 @@
 
-  beforeEach(() => {
+  beforeEach(fakeAsync(() => {
     fixture = TestBed.createComponent(InputGoogleAutocompleteComponent);
     component = fixture.componentInstance;
+
+    component.predictionList = predictionList;
+    component.autocompleteService = mockAutocompleteServiceInstance;
+    component.initPredictList();
     fixture.detectChanges();
-  });
+  }));
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)